hip-service/OTP/OtpRepository.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using hip_service.Link.Patient.Models;
+using hip_service.OTP.Models;
+
+namespace hip_service.OTP
+{
+    public class OtpRepository: IOtpRepository
+    {
+        private readonly LinkPatientContext _linkPatientContext;
+        public OtpRepository(LinkPatientContext linkPatientContext)
+        {
+            _linkPatientContext = linkPatientContext;
+        }
+        public async Task<Tuple<OtpRequest, Exception>> SaveOtpRequest(string otp, string sessionId)
+        {
+            var dateTimeStamp = new DateTime (DateTime.Today.Year, DateTime.Today.Month, DateTime.Today.Day, 
+                    DateTime.Now.Hour, DateTime.Now.Minute, DateTime.Now.Second).ToUniversalTime().
+                ToString("yyyy-MM-ddTHH:mm:ssZ");
+
+            var otpRequest = new OtpRequest(sessionId, dateTimeStamp, otp);
+            _linkPatientContext.OtpRequests.Add(otpRequest);
+            try
+            {
+                await _linkPatientContext.SaveChangesAsync();
+                return new Tuple<OtpRequest, Exception>(otpRequest, null);
+            }
+            catch (Exception exception)
+            {
+                Console.WriteLine(exception);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4769144)

Reply:
done